misc/python/materialize/parallel_workload/parallel_workload.py

```diff
@@ -354,13 +354,14 @@ def run(
         for worker, thread in zip(workers, threads):
             if thread.is_alive():
                 print(
-                    f"{thread.name} still running ({worker.exe.mz_service}): {worker.exe.last_log}"
+                    f"{thread.name} still running ({worker.exe.mz_service}): {worker.exe.last_log} ({worker.exe.last_status})"
                 )
         print_stats(num_queries, workers, num_threads)
         if num_threads >= 50:
             # Under high load some queries can't finish quickly, especially UPDATE/DELETE
             os._exit(0)
         print("Threads have not stopped within 5 minutes, exiting hard")
+        time.sleep(1000000)
```

Comment:
Why?

Reply:
Thanks for catching, that was for my local testing only